src/dialogs/more-info/controls/more-info-sun.ts

```diff
@@ -73,9 +73,6 @@ class MoreInfoSun extends LitElement {
         display: inline-block;
         white-space: nowrap;
       }
-      ha-relative-time::first-letter {
```

Comment:
I think this needs to stay since here the relative time is used in conjunction with text before it.

![image](https://user-images.githubusercontent.com/114137/136182091-c34e47c6-7b35-4cd8-8368-84f98b40879f.png)

Reply:
It will be lower case